framework/platform/platform.cpp

```diff
@@ -153,6 +154,28 @@ void Platform::run()
 	}
 }
 
+std::unique_ptr<RenderContext> Platform::create_render_context(Device &device, VkSurfaceKHR surface) const
+{
+	auto context = std::make_unique<RenderContext>(device, surface, window->get_width(), window->get_height());
+
+	context->set_surface_format_priority({{VK_FORMAT_R8G8B8A8_SRGB, VK_COLOR_SPACE_SRGB_NONLINEAR_KHR},
+	                                      {VK_FORMAT_B8G8R8A8_SRGB, VK_COLOR_SPACE_SRGB_NONLINEAR_KHR},
```

Comment:
These are in a different order to the original code.

Reply:
The surface priority was moved for convenience with the presentation modes. I missed the fact that one prioritises UNORM and the other sRGB. I'll fix this later today :)